dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -693,11 +696,11 @@ private void addBulkRequest(final BulkIndexWrapper bulk, final List<Contentlet>
                 if (this.isWorking(contentlet)) {
                     mapping = gson.toJson(mappingAPI.toMap(contentlet));
                     if (!forReindex || info.reindex_working == null) {
-                        bulk.add(new IndexRequest(info.working, "content", id)
+                        bulk.add(new IndexRequest(info.working, "_doc", id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "_doc" appears 9 times in this file; the first occurrence is on line 690](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)